app/models/spotlight/blacklight_configuration.rb

```diff
@@ -319,7 +319,7 @@ def set_index_field_defaults(field)
       return unless index_fields.blank?
 
       views = default_blacklight_config.view.keys | [:show, :enabled]
-      field.merge! Hash[views.map { |v| [v, !title_only_by_default?(v)] }]
+      field.merge! Hash[views.map { |v| [v, field.send(v).nil? ? !title_only_by_default?(v) : field.send(v)] }]
```

Comment:
Is `field.merge! (views - field.keys).map { |v| [v, !title_only_by_default?(v)] }.to_h` any more satisfying/clear?

Reply:
Yeah, I like that a bit better.  Will update.  Thanks.